jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/SignalStatusPackageListBuilder.java

```diff
@@ -14,22 +14,25 @@ private SignalStatusPackageListBuilder() {
     public static J2735SignalStatusPackageList genericSignalStatusPackageList(JsonNode sigStatus) {
         J2735SignalStatusPackageList signalStatusPackageList = new J2735SignalStatusPackageList();
 
-        if (sigStatus.isArray()) {
-			Iterator<JsonNode> elements = sigStatus.elements();
-
-			while (elements.hasNext()) {
-				signalStatusPackageList.getSigStatus()
-                    .add(SignalStatusPackageBuilder.genericSignalStatusPackage(elements.next()));
-			}
-		} else {
-			JsonNode signalStatusPackage = sigStatus.get("SignalStatusPackage");
-			if(signalStatusPackage != null)
-			{
-				signalStatusPackageList.getSigStatus()
-                	.add(SignalStatusPackageBuilder.genericSignalStatusPackage(signalStatusPackage));
-			}
-
-		}
+        JsonNode signalStatusPackage = sigStatus.get("SignalStatusPackage");
+        if(signalStatusPackage != null)
+        {
+            if (signalStatusPackage.isArray()) {
+                Iterator<JsonNode> elements = signalStatusPackage.elements();
+    
+                while (elements.hasNext()) {
+                    signalStatusPackageList.getSigStatus()
+                        .add(SignalStatusPackageBuilder.genericSignalStatusPackage(elements.next()));
+                }
+            } else {
+                signalStatusPackageList.getSigStatus()
+                        .add(SignalStatusPackageBuilder.genericSignalStatusPackage(signalStatusPackage));
+    
+            }
+        }
+        else {
+            return null;
+        }
```

Comment:
suggestion(non-blocking): invert the if to pull the main code flow out of the if statement and make it clearer to the reader that we return a null value if there isn't a JsonNode for "SignalStatusPackage".
```suggestion
        if (signalStatusPackage == null) {
            return null;
        }
        
        if (signalStatusPackage.isArray()) {
            Iterator<JsonNode> elements = signalStatusPackage.elements();

            while (elements.hasNext()) {
                signalStatusPackageList.getSigStatus()
                        .add(SignalStatusPackageBuilder.genericSignalStatusPackage(elements.next()));
            }
        } else {
            signalStatusPackageList.getSigStatus()
                    .add(SignalStatusPackageBuilder.genericSignalStatusPackage(signalStatusPackage));

        }
```

Reply:
I reversed the if statement as you requested and applied the same thing to the SignalStatusListBuilder.